src/qml/qgismobileapp.qml

```diff
@@ -3621,22 +3618,21 @@ ApplicationWindow {
 
       LayerLoginDialog {
         id: loginDialog
-
         anchors.fill: parent
-
         visible: true
-
-        realm: loginDialogPopup.realm
         inCancelation: false
 
-        onEnter: {
-          qfieldAuthRequestHandler.enterCredentials( realm, usr, pw)
+        property string realm: ""
+
+        onEnter: ( usr, pw ) => {
+          qfieldAuthRequestHandler.enterCredentials( loginDialog.realm, usr, pw)
```

Comment:
```suggestion
        onEnter: (username, password) => {
          qfieldAuthRequestHandler.enterCredentials( loginDialog.realm, username, password)
```

I know it used to be like this, but since you are on it, would be great to have proper names :)